bundles/org.openhab.binding.somfytahoma/README.md

```diff
@@ -124,11 +128,19 @@ When a roller shutter-like thing receives STOP command, there are two possible b
 If you want to set the MY position of a roller shutter and you don't care the possible movement, try sending the MOVE command (OH2 does not know MY, so it stands for "move to MY position")
 
 ```
-sendCommand(ROLLERSHUTTER_THING, MOVE)
+sendCommand(CONTROL_CHANNEL, MOVE)
 ```
 or
 ```
-ROLLERSHUTTER_THING.sendCommand(MOVE)
+CONTROL_CHANNEL.sendCommand(MOVE)
+```
+Blinds and adjustable slats roller shutters can control their closure and orientation by sending a comma separated string consisting of closure (0-100) and orientation (0-100) to the "closure_orientaion" channel.
+```
+sendCommand(CLOSURE_ORIENTATION_CHANNEL, “50,50”)
```

Comment:
See above.

Pleas add an empty line between text and examples.

Reply:
Fixed